zebra/zebra_rib.c

```diff
@@ -4319,6 +4316,7 @@ struct route_entry *zebra_rib_route_entry_new(vrf_id_t vrf_id, int type,
 
 void zebra_rib_route_entry_free(struct route_entry *re)
 {
+	zapi_re_opaque_free(re->opaque);
```

Comment:
could we NULL that pointer too? it looks like there aren't a lot of callers to that function - maybe it could just take the re and do what's needed?

Reply:
Sure, fixed :+1: